src/MSALWrapper/MSALWrapper.csproj

```diff
@@ -30,10 +30,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.4" />
-    <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.19.6" />
+    <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.32.0" />

```

Comment:
This doesn't have any impact on the `azureauth ado pat` subcommand and the associated PAT caching?

Reply:
I haven't tested

Reply:
I have tested and this shouldn't cause any issues.